test/journeys/lib/test-helpers/space-widget/meet.js

```diff
@@ -4,6 +4,8 @@ import {switchToMeet} from './main';
 import {getEventLog} from '../../events';
 import {constructHydraId} from '../../hydra';
 
+export const FEATURE_FLAG_GROUP_CALLING = `js-widgets-group-calling`;
```

Comment:
With more and more feature flags, should we create a package that just has the list of feature flags?

Reply:
We probably should come up with some way to track feature flags. If it is a regular package, we need to make sure it is available on npm since the journey tests don't get from the node modules though.

Reply:
On thing I just realized this morning. You could reference packages just using relative paths... lot of ‘../../../‘ but that would work